pennylane/transforms/split_tape.py

```diff
@@ -374,32 +266,72 @@ def processing_fn(expanded_results):
 
         return results[0] if len(results) == 1 else results
```

Comment:
I think here we should return a tuple at least for the new return type.

Reply:
Changed to tuple! 👍